.github/workflows/add-users-everyone-github-team.yml

```diff
@@ -13,7 +13,7 @@ jobs:
         with:
           python-version: "3.10"
       - run: python3 -m pip install -r scripts/python/requirements.txt
-      - run: python3 scripts/python/add-users-everyone-github-team.py ${{ secrets.OPERATIONS_ENGINEERING }}
+      - run: python3 scripts/python/add_users_everyone_github_team.py ${{ secrets.OPERATIONS_ENGINEERING }}
```

Comment:
rename was needed to make the file importable in python. Ref: https://visualgit.readthedocs.io/en/latest/pages/naming_convention.html#modules

Reply:
good spot, I noticed it the other day using a lint and didn't spot it. Good chance all the scripts needs that applied to them